views/css/jobRolesStyle.css

```diff
@@ -20,6 +20,17 @@ h1{
     filter: contrast(0.9);
 }
 
+
+@keyframes spinCursor {
+    0% { transform: rotate(0deg); }
+    100% { transform: rotate(360deg); }
+  }
+  
+  .spinning-cursor {
```

Comment:
We are missing pictures of the UI and how the spinning-cursor looks on the page.

Reply:
Added to the ticket